bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +34,17 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = InvalidLinkBear.get_status_code_or_error(link, timeout)
                 yield line_number + 1, link, code
 
-    def run(self, filename, file):
+    def run(self, filename, file, timeout: int=2):
         '''
         Yields results for all invalid links in a file.
+
+        :param timeout: Request time out period (Default: 2 sec).
```

Comment:
And you don't need to document the default, I think we grab the value automatically via the signature and pass that to the help :+1: